lib/matplotlib/transforms.py

```diff
@@ -2355,8 +2354,6 @@ def __init__(self, a, b, **kwargs):
         self._b = b
         self.set_children(a, b)
 
-    is_affine = property(lambda self: self._a.is_affine and self._b.is_affine)
-
```

Comment:
I suspect this should *not* be removed?

Reply:
This was a duplicate definition. See transforms.py l. 2394.